app/scripts/controllers/landingPage.js

```diff
@@ -84,11 +86,16 @@ angular.module('openshiftConsole')
       }
     }
 
-    function resetRouteParams() {
-      // Reset the route params in the next digest cycle
-      $timeout(function() {
-        $location.replace();
-        $location.search('startTour', null);
-      });
-    }
+    var deregisterLocationStart = $scope.$on('$locationChangeStart', function(event, newUrl) {
+      if (newUrl.indexOf('startTour') > 0) {
+        event.preventDefault();
```

Comment:
Ah OK, understood. I'd add a comment saying this is to prevent the URL from changing in the location bar.